neural_compressor/compression/pruner/model_slim/README.md

```diff
@@ -51,7 +51,7 @@ Auto Slim
 # auto slim config
 # part1 generate pruning configs for the second linear layers. 
 pruning_configs = []
-from neural_compressor.compression.pruner.model_slim import parse_auto_slim_config
+from neural_compressor.compression.pruner.model_slim.auto_slim import parse_auto_slim_config

```

Comment:
the import path is too long, how about neural_compressor.compression.pruner.parse_auto_slim_config

Reply:
current we use from neural_compressor.compression.pruner import parse_auto_slim_config